tasks/configure_new.yml

```diff
@@ -0,0 +1,67 @@
+---
+- name: PGBouncer | Create database conf directory
+  file:
+    path: "{{ pgbouncer_conf_dir }}"
+    state: directory
+    recurse: yes
+
+- name: PGBouncer | Create auth file
+  copy:
+    dest: "{{ pgbouncer_auth_file }}"
+    content: ""
+    force: no
+
+- name: PGBouncer | Configure database conf
+  template:
+    src: "database.ini.j2"
+    dest: "{{ pgbouncer_conf_dir }}/{{ item.name }}.ini"
+  with_items: "{{ pgbouncer_database_aliases | default([]) }}"
+  tags:
+    - deploy
+
+- name: PGBouncer | get conf file list
+  command: ls "{{ pgbouncer_conf_dir }}"
+  register: pgbouncer_database_conf_list
+  tags:
+    - deploy
+
+- set_fact:
+    pgbouncer_database_conf_list: "{{ pgbouncer_database_conf_list.stdout_lines }}"
```

Comment:
if --deploy tag will be given this fact will not be set.  you may see unexpected result for var `pgbouncer_database_conf_list`